<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecialMagicDialog</class>
 <widget class="QDialog" name="SpecialMagicDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>308</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>308</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>308</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Special / Magic List</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="specialLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="2" column="0">
      <widget class="QCheckBox" name="sNeutral">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>----</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QCheckBox" name="sEarthAttack">
       <property name="text">
        <string>Earth Attack</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="sFleshToStone">
       <property name="text">
        <string>Flesh to Stone</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="sAbsorbMana">
       <property name="text">
        <string>Absorb Mana</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="sAbsorbHP">
       <property name="text">
        <string>Absorb HP</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="sWaterLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/Water13.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="sAllEnemies">
       <property name="text">
        <string>All Enemies</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="sPoison">
       <property name="text">
        <string>Poison</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="sCritical">
       <property name="text">
        <string>Critical</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="sAirAttack">
       <property name="text">
        <string>Air Attack</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="sBreakLegs">
       <property name="text">
        <string>Break Legs</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="sRendWings">
       <property name="text">
        <string>Rend Wings</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="sFireLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/Fire13.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QCheckBox" name="sWaterAttack">
       <property name="text">
        <string>Water Attack</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="sEarthLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/Earth13.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sNoneLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/None13.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="sSleep">
       <property name="text">
        <string>Sleep</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="sFireAttack">
       <property name="text">
        <string>Fire Attack</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="sDestroyMana">
       <property name="text">
        <string>Destroy Mana</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="sAirLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/Air13.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="sSpecialLabel">
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Special</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="magicLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="2" column="1">
      <widget class="QCheckBox" name="mAgnis">
       <property name="text">
        <string>Agnis</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="mAdSlahm">
       <property name="text">
        <string>Ad Slahm</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="mEarthLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/Earth13.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QCheckBox" name="mVahli">
       <property name="text">
        <string>Vahli</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="mVahlis">
       <property name="text">
        <string>Vahlis</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QCheckBox" name="mAdRoqua">
       <property name="text">
        <string>Ad Roqua</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="mMaltia">
       <property name="text">
        <string>Maltia</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="mUlvis">
       <property name="text">
        <string>Ulvis</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QCheckBox" name="mSelahm">
       <property name="text">
        <string>Selahm</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="mWaterLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/Water13.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="mAgni">
       <property name="text">
        <string>Agni</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="mMedinia">
       <property name="text">
        <string>Medinia</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="mDevaAgni">
       <property name="text">
        <string>Deva Agni</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="mMaltis">
       <property name="text">
        <string>Maltis</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QCheckBox" name="mUlvia">
       <property name="text">
        <string>Ulvia</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mAirLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/Air13.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QCheckBox" name="mDevaMalti">
       <property name="text">
        <string>Deva Malti</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QCheckBox" name="mDevaVali">
       <property name="text">
        <string>Deva Vali</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="mUlvi">
       <property name="text">
        <string>Ulvi</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="mDevaUlvi">
       <property name="text">
        <string>Deva Ulvi</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="mAdVenon">
       <property name="text">
        <string>Ad Venon</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QCheckBox" name="mMedina">
       <property name="text">
        <string>Medina</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="mAdMumuls">
       <property name="text">
        <string>Ad Mumuls</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="mSelahmVenon">
       <property name="text">
        <string>Selahm Venon</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mFireLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/Fire13.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="mAgnia">
       <property name="text">
        <string>Agnia</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QCheckBox" name="mVahlia">
       <property name="text">
        <string>Vahlia</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="mMalti">
       <property name="text">
        <string>Malti</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="mSelSelahm">
       <property name="text">
        <string>Sel Selahm</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="mMagicLabel">
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Magic</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
